docs/api/conf.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# nf-core/tools documentation build configuration file, created by
+# sphinx-quickstart on Sun Oct 21 19:28:42 2018.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, '../..')
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autodoc',
+    'sphinx.ext.githubpages',
+    'sphinx.ext.napoleon']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
```

Comment:
Can we used markdown instead please? We use it everywhere else so better to keep consistency if possible..

Reply:
For the index file sure, this is the entry point of the documentation and will not be touched by the automatic creation of documents. For the auto-generated API doc, this is `.rst`. But as this is the part not touched by us anyway (as it is generated automatically from the Python docstrings), I do not see a problem. The final output will be HTML anyway, so I think we can overlook the intermediary `rst` file from the autodocs?

Reply:
What do you think?

Reply:
Ah ok cool - if it's the auto-generated doc and never manually edited then I guess it doesn't matter..